src/NextGen/DonationForm/Models/DonationForm.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Give\NextGen\DonationForm\Models;
+
+use DateTime;
+use Give\Framework\FieldsAPI\Form;
+use Give\Framework\Models\Contracts\ModelCrud;
+use Give\Framework\Models\Contracts\ModelHasFactory;
+use Give\Framework\Models\Model;
+use Give\NextGen\DonationForm\Factories\DonationFormFactory;
+use Give\NextGen\DonationForm\Traits\DonationFormCRUD;
+use Give\NextGen\DonationForm\ValueObjects\DonationFormStatus;
+
+/**
+ * @unreleased
+ *
+ * @property int $id
+ * @property string $formTitle
+ * @property DateTime $createdAt
+ * @property DateTime $updatedAt
+ * @property DonationFormStatus $status
+ * @property array $settings
+ * @property string $blocksContent
+ * @property Form $schema
+ */
+class DonationForm extends Model implements ModelCrud, ModelHasFactory
+{
+    use DonationFormCRUD;
+
+    /**
+     * @inheritdoc
+     */
+    protected $properties = [
+        'id' => 'int',
+        'formTitle' => 'string',
+        'createdAt' => DateTime::class,
+        'updatedAt' => DateTime::class,
+        'status' => DonationFormStatus::class,
+        'settings' => 'array',
+        'blocksContent' => 'string',
```

Comment:
Didn't we say this will be a `BlocksCollection` type?

Reply:
Resolved by https://github.com/impress-org/givewp-next-gen/pull/63/commits/5122a3ce7bfe4044bab8c5b7ac1c3cd914df7340.

Reply:
Thanks! i'm thinking we just call this property blocks since it's now a collection of blocks, `$form->blocks`

Reply:
Ah, yeah. That works.

Reply:
Resolved by https://github.com/impress-org/givewp-next-gen/pull/63/commits/c954c2eaf8c92ead88ad687d7ba52396deeda33e.